Build for Ubuntu 23.04 and Debian 12

With Debian 12 and Ubuntu 23.04 - the dependency `libwxgtk3.0-gtk3-0v5` does not exist anymore.
It is replaced by `libwxgtk3.2-dev`

Would it be possible to provide a release for this too?
I somehow had problems building it myself changing the `src/CMakeLists.txt` 

```
-- DEPS_INSTALL_DIR : /home/admin/fh-gitlab/projects/Antares_Simulator/src/../../rte-antares-deps-release
CMake Error at /usr/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find Boost (missing: Boost_INCLUDE_DIR)
Call Stack (most recent call first):
  /usr/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.25/Modules/FindBoost.cmake:2376 (find_package_handle_standard_args)
  CMakeLists.txt:230 (find_package)
```

--- a/README.md
+++ b/README.md
@@ -22,6 +22,8 @@ Until 2018 it was distributed under the terms of a proprietary license.
 
 In May 2018 RTE decided to release the project under the GPLv3 license.
 
+The GUI is deprecated in favor of [Antares Web](https://github.com/AntaresSimulatorTeam/AntaREST).
+
 [linux_system_svg]: https://github.com/AntaresSimulatorTeam/Antares_Simulator/workflows/Linux%20CI%20(system%20libs)/badge.svg
 
 [linux_system_link]: https://github.com/AntaresSimulatorTeam/Antares_Simulator/actions?query=workflow%3A"Linux%20CI%20(system%20libs)"
@@ -40,6 +42,7 @@ In May 2018 RTE decided to release the project under the GPLv3 license.
 - Antares web site :  https://antares-simulator.org
 - RTE web site  : http://www.rte-france.com/
 - Doxygen code documentation  : https://antaressimulatorteam.github.io/Antares_Simulator/
+- Antares Web : https://github.com/AntaresSimulatorTeam/AntaREST
 
 
 # Installation

